freqtrade/rpc/telegram.py

```diff
@@ -447,13 +447,13 @@ def _status(self, update: Update, context: CallbackContext) -> None:
                     ("` (since {open_date_hum})`" if r['is_open'] else ""),
                     "*Current Pair:* {pair}",
                     "*Amount:* `{amount} ({stake_amount} {base_currency})`",
-                    "*Buy Tag:* `{buy_tag}`" if r['buy_tag'] else "",
-                    "*Sell Reason:* `{sell_reason}`" if r['sell_reason'] else "",
+                    "*Entry Tag:* `{buy_tag}`" if r['buy_tag'] else "",
+                    "*Exit Reason:* `{sell_reason}`" if r['sell_reason'] else "",
```

Comment:
hah - i added some tests for this :smile:

Reply:
Ah okay, that's why it failed. I didn't do the unit tests because I assumed it weren't something that gonna be tested. Modifying the tests rn

Reply:
not "extensively" - they're still necessary to make sure it's not accidentally broken by some other changes.
That's especially true for RPC methods - where it would be strange/odd if these messages disappeared for some odd reason.